packages/main/src/plugin/color-registry.ts

```diff
@@ -1182,6 +1182,13 @@ export class ColorRegistry {
       dark: colorPalette.gray[900],
       light: colorPalette.gray[100],
     });
+
+    // contrast color for the other status colors,
+    // e.g. to use in status icons
+    this.registerColor(`${status}contrast`, {
+      dark: colorPalette.white,
+      light: colorPalette.white,
+    });
```

Comment:
not sure why to define a color if it's always white ?

Reply:
Because it was still a design choice. If someone else was making a theme they might be ok with less contrast (maybe an off white), something else might work better with their status colors, or could go transparent.